tfprotov5/tftypes/list.go

```diff
@@ -6,6 +6,7 @@ import "fmt"
 // of the same type.
 type List struct {
 	ElementType Type
+	_           []struct{}
```

Comment:
can you please comment why these are added some someone doesn't inadvertently remove them.

this also prevents usage in map keys i believe as well, correct?

Reply:
Yes, Types can no longer be map keys after this change.

Reply:
Nor can they be used in switch statements, or anywhere else that relies on some concept of comparison or uniqueness.